conftest.py

```diff
@@ -21,7 +21,7 @@ def pytest_addoption(parser):
         help="If provided, will not annihilate the data generated.")
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture()
 def workspace(request):
     # TODO: Use pytest.config.option('--workspace')
     test_workspace = tempfile.mkdtemp()
```

Comment:
the workspace no longer has module-scope either, so each fixture gets a new directory, which is obliterated when the calling fixture exits.